### DISCUSSION TITLE
Does pandapower Support dynamic simulation ???????

### DISCUSSION THREAD

**Discussion by Adarsh-365** [e2nIEE/pandapower#2116] [Q&A] [answered]
so i am trying to create dynamic simulation Load Frequency Control (LFC) scenario
i have created grid network in Pandapower and i want to

1)Add Load of 1000Mw due to Load imbalance causing frequency will fall
2) Then i will Remove that load then frequency will maintain

does it possible to do in pandapower

**Comment by vogt31337** [comment, marked as answer]
Sadly no, pandapower does not simulate such dynamic processes. You could have a look into dynawo.
